routes/candidateRoutes.js

```diff
@@ -47,7 +47,7 @@ candidateRouter.get('/id=:id', authorizedOrdinaryUser, async (req, res) => {
     }
 });
 
-candidateRouter.get('/all', async (req, res) => {
+candidateRouter.get('/all', authorizedOrdinaryUser, async (req, res) => {
```

Comment:
This was not a mistake, it was intentionally left so. Candidates information are usually public.

Reply:
Okay, I thought it was my mistake. Agreed